src/types/git.ts

```diff
@@ -0,0 +1,348 @@
+/* ---------------------------------------------------------------------------------------------
```

Comment:
Since this is a vendored file we should have a comment mentioning the fact and also its source, along with any changes vs the source. See https://github.com/cytechmobile/radicle-vscode-extension/blob/main/src/webviews/src/assets/codicon.css#L1-L11 for how it's already handled in the repo.